### DISCUSSION TITLE
Installer: GNU/Linux 11 (bullseye) is unsupported

### DISCUSSION THREAD

**Discussion by Fahmula** [RaspAP/raspap-webgui#1054] [Q&A] [answered]
I had install RaspAP a few weeks ago with no issues but now I'm getting a error stating "Raspbian GNU/Linux 11 (bullseye) is unsupported. Please install on a supported distro"

The installation continues despite the error but when its competes, I can't access the webgui
![RaspAP](https://user-images.githubusercontent.com/55062259/141815994-178ea8c5-ebbd-4233-94b2-2f39c6a775c9.png)
.
![RaspAP2](https://user-images.githubusercontent.com/55062259/141816003-6fa07b3c-6168-49ad-ab6e-6bd9179b05eb.png)

**Comment by billz** [comment, marked as answer]
New release with Debian 11 / latest RPi OS support was published today. Try the installer again, thanks

https://github.com/RaspAP/raspap-webgui/releases/tag/2.6.8

**Reply by Fahmula** [reply]
Thanks! it's working now.
